packages/webpack-bundler-runtime/src/container.ts

```diff
@@ -0,0 +1,271 @@
+import bundler_runtime_base from './index';
+import type { Options } from '@module-federation/runtime/types';
+
+interface ExposesConfig {
+  /**
+   * Request to a module that should be exposed by this container.
+   */
+  import: ExposesItem | ExposesItems;
+  /**
+   * Custom chunk name for the exposed module.
+   */
+  name?: string;
+}
+
+type Exposes = (ExposesItem | ExposesObject)[] | ExposesObject;
+/**
+ * Module that should be exposed by this container.
+ */
+type ExposesItem = string;
+/**
+ * Modules that should be exposed by this container.
+ */
+type ExposesItems = ExposesItem[];
+
+interface ExposesObject {
+  /**
+   * Modules that should be exposed by this container.
+   */
+  [k: string]: ExposesConfig | ExposesItem | ExposesItems;
+}
+
+interface ExtendedOptions extends Options {
+  exposes: { [key: string]: () => Promise<() => any> };
+}
+
+export const createContainer = async (federationOptions: ExtendedOptions) => {
+  // await instantiatePatch(federationOptions, true);
```

Comment:
Commented out, used to support import maps, but should be implemented via sdk/import-maps instead of a patch on function

Reply:
Removing patch since something else should be cone for async container / initialization.